### PR TITLE
[Editor] Folder icons for Data Explorer

### PR DESCRIPTION
**Pull request by BreakfastBrainz2**

### CODE REVIEW

File: FrostyEditor/FrostyEditor.csproj [hunk 1]
```diff
@@ -7,6 +7,20 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="Assets\FolderCollapsed.png" />
+    <None Remove="Assets\FolderExpanded.png" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ApplicationDefinition Include="Assets\FolderCollapsed.png" />
```
Comment (wannkunstbeikor) [inline FrostyEditor/FrostyEditor.csproj]: any reason why this is also an applicationdefinition?

File: FrostyEditor/ViewModels/Tools/DataExplorerViewModel.cs [hunk 2]
```diff
@@ -11,13 +15,35 @@ namespace FrostyEditor.ViewModels.Tools;
...
+
+    private static FolderIconConvert? s_folderIconConverter;
+
     [ObservableProperty]
     private string m_test = "Explorer";
 
     [ObservableProperty]
     private FlatTreeDataGridSource<AssetModel> m_assetsSource;
-    
-    public HierarchicalTreeDataGridSource<FolderTreeNodeModel> FolderSource { get; }
```
Comment (wannkunstbeikor) [inline FrostyEditor/ViewModels/Tools/DataExplorerViewModel.cs:20]: why is this removed?

Comment (wannkunstbeikor) [reply FrostyEditor/ViewModels/Tools/DataExplorerViewModel.cs:20]: nvm im stupid

File: FrostyEditor/FrostyEditor.csproj (hunk 1 above)
Comment (BreakfastBrainz2) [reply FrostyEditor/FrostyEditor.csproj]: not sure, it does that when i set the resource type to AvaloniaResource

Comment (wannkunstbeikor) [reply FrostyEditor/FrostyEditor.csproj]: weird
